frontend/src/pages/Player/Toolbar/NewCommentForm/CommentTextBody/CommentTextBody.tsx

```diff
@@ -58,6 +61,14 @@ const CommentTextBody = ({
 		}
 	}, [shouldAutoFocus])
 
+	useEffect(() => {
+		const parts = commentText.split('@')
+		const last = parts.at(-1)
+		if (last) {
+			setLatestAt(`@${last}`)
+		}
+	}, [commentText])
```

Comment:
Wondering if we need to be using state to calculate this or if it would be better to just derive `latestAt` on each render 🤔 - I think I'd go that route unless we noticed a performance issue or something that we're trying to work around.